src/main.py

```
@@ -315,8 +319,20 @@ def main():
 
             elif prompt.startswith("cd"):
                 entered_dir = prompt.replace("cd ", "")
-                os.chdir(entered_dir)
-            
+                try:
+                    os.chdir(entered_dir)
+                except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)